src/server_manager/web_app/ui_components/outline-server-settings.ts

```diff
@@ -253,6 +260,13 @@ Polymer({
               always-float-label=""
               maxlength="100"
             ></paper-input>
+            <a
+              class="documentation-link"
+              href="https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/shadowbox/server/api.yml"
```

Comment:
Can we create a page to host the API documentation? It can be in the Wiki.
I'd rather point to a page we control, so we can change and provide more context.

It would be helpful to perhaps add some examples, like https://github.com/Jigsaw-Code/outline-server/tree/master/src/shadowbox#access-keys-management-api.

Reply:
Another PR sounds good. BTW, we just need a simple page that links to the resource and gives some context.